addons/mail/static/src/discuss/call/web/discuss_patch.js

```diff
@@ -1,11 +1,28 @@
 /* @odoo-module */
 
 import { Discuss } from "@mail/core/common/discuss";
+import { useEffect } from "@odoo/owl";
 
 import { patch } from "@web/core/utils/patch";
 
 patch(Discuss.prototype, "discuss/call/web", {
+    setup(...args) {
+        this._super(...args);
+        Object.assign(this.state, { openInviteButton: 0 });
+        useEffect(
+            () => {
+                if (this.state.openInviteButton === 0) {
+                    return;
+                }
+                this.threadActions.actions.find((action) => action.id === "add-users")?.onSelect();
+            },
+            () => [this.state.openInviteButton]
+        );
+    },
     onStartMeeting() {
-        this.threadActions.actions.find((action) => action.id === "add-users").onSelect();
+        if (owl.status(this) === "destroyed") {
+            return;
+        }
```

Comment:
probably not necessary, assuming `useEffect` does not execute if the component is destroyed?

Reply:
`useEffect` is always called after patching and mount, so cannot be called on destroyed component